documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -92,6 +93,7 @@ We request OpenAPI(Swagger) spec authoring be assigned to engineers who have an
 | M2062	| RequiredReadOnlyPropertiesValidation | Property '{0}' is a required property. It should not be marked as 'readonly'. | Error |
 | M2054	| SecurityDefinitionsStructureValidation | An OpenAPI(swagger) spec must have security definitions and must adhere to the specific structure. | Error |
 | M2022	| XmsExamplesProvidedValidation | Please provide x-ms-examples describing minimum/maximum property set for response/request payloads for operations.{0} | Error |
+| S2006	| ControlCharactersNotAllowed | Specification must not contain any control characters.
```

Comment:
the rules that start with "S" should be updated, not just documentation wise but code wise to be in the "R" pattern.